api/src/middleware/assessmentsRouter.ts

```diff
@@ -0,0 +1,45 @@
+import { Router } from 'express';
+import { itemEnvelope } from './responseEnvelope';
+
+const assessmentsRouter = Router();
+assessmentsRouter.get('/', (req, res) => {
+  const response = { behaviour: 'list of assessments' };
```

Comment:
Let's start each behavior string with a verb. This will help people new to the backend next week understand each route.